### ISSUE TITLE
[ACCESSIBILITY]: Update e2e documentation with axeCheck best practices

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#20119]
## Issue Description
Now that we've (almost?) [shipped the axe-core 4.1.1 update](https://github.com/department-of-veterans-affairs/vets-website/pull/16056), I'd like to write best practice documentation for e2e accessibility testing.

---
## Tasks
- [ ] Update the [Cypress best practices on VSP](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-best-practices-on-vsp.md) document with a new section on "what makes a good accessibility test in Cypress"

## Acceptance Criteria
The documentation updates should include the following:
- [ ] Pages should be fully rendered before an axe check is run
- [ ] Consider axe checks for forms before they are filled out, and after
- [ ] Consider opening accordions and / or modal dialogs and running a second axe check
- [ ] Re-enable the color-contrast check for 1-2 tests per product

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by pjhill** [comment]
We may be able to add some of these suggestions [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/A11y-Testing.1935409178.html).
